src/main.js

```diff
@@ -207,7 +202,8 @@ async function messageReceived(message) {
 	}
 
 	//*USES X*
-	if ((lower.startsWith('*uses ') || lower.startsWith('_uses ')) && (lower.endsWith('*') || lower.endsWith('_'))) {
+	const pokemonRegex = /^\*uses .+\*|_uses .+_$/;
```

Comment:
```suggestion
	const pokemonRegex = /^(\*uses .+\*|_uses .+_)$/;
```
need the () inside ^ and $

Reply:
![image](https://user-images.githubusercontent.com/16765522/70441154-66f58780-1a59-11ea-88ee-0bebded39e64.png)

Reply:
@connertennery this is correct behavior. Notice the `i` flag is missing at the end, that's because bucket evaluates the regex on `message.content.toLowerCase()`

Reply:
@clee-dev Right, I remembered it was lowercase, I was just putting my tests in here. Not saying any part of it was wrong.